presto-spi/src/main/java/com/facebook/presto/spi/type/UnscaledDecimal128Arithmetic.java

```diff
@@ -90,6 +93,10 @@
             throw new RuntimeException(e);
         }
 
+        for (int i = 0; i < POWERS_OF_FIVE.length; ++i) {
+            POWERS_OF_FIVE[i] = unscaledDecimal(BigInteger.valueOf(5).pow(i));
```

Comment:
but it doesn't matter, it's executed only once and `BigInteger` version is more reliable